DogeNet.IdentityServer/tempkey.jwk

```diff
@@ -0,0 +1 @@
+{"alg":"RS256","d":"g09VXgZ_UB62pPnxOdzg5X9sFZ69Z_IXEOkGhpIfMURoGmpAr4omGpecsq0TLLiQDATq4jz0ABMJWcPwjwxdakqg-lkL-dgRAxM8ls6fgGkTRMmCBH-id05NJUKP-v5QLeFKydERRdW1OTrRUUq0pJCCmgJbPXOmfWWncD3J0ejhbs13ztS5XmoOlKllcOuwLl0_Be4-QYUZj-gkhexk5DEhaAqVFENZm9ChUdy3iIGQdklBEPqpofRz9ZhybzbGPuK8hBsRUSHWrTSrAixQGRFpCT-pgzsx5K1Dl8N2JQq8qNdw029-Ww8ZQlD3JRuv5CmxqtSdcCvy6Abqk3xjoQ","dp":"D2hGCkkBKb3DZ3mS68QNmPliMSuaLSM7VNm8zh1RNi54xYdTGZgvb2Hq9YtveiHfnqRgsnVd5uInSRVIy2OQGQ-F80RMT6nofsK_FvraCBmHxHV6xMr63FN3Z0HhK2KxGlATTtiPaW6bXrAtYdKSOZp1ZLYqjqBK4Bo1sGLOz60","dq":"Hz2k_2ILj2CKNeCg3Fc_CksnYRD8uPF5wSXot4fXujBM16dz6yPHA6me9kfm3uaEFmla3StulFyoOzR-Jf20bbOXOtKHRjhmoXrGZWfZgGxNP3WZox7bgjZVhZSNYS-eIMC_caoOPHaD1Eg749bLZ4KlVVmxckr1j_WGTMSwklc","e":"AQAB","kid":"4FED10D77BC20CBCD8337CAEAABF5E7F","kty":"RSA","n":"qek87HLTG-5ndCzqThzYQ3Li87jEZAk24wIE2auEm0rQPTtgKm5_4edzqIvGthoa9BizaPiNBou2og24M2tCOjADlWFxMeDB6AClMPGYpA0zYh8rihjc5W_9WWcEUHV7gzjbOBBPsZbsI_z67QPzyN8Ut0fhl1iddn-PNqfHeBMJ88G4fCceDUYnXTsMRzSxQtGigcMi96-k5wlcJNB-yp5830AfnR4ZgA-TUzPbewRGTvIFT3J4aX3nXWuFWCFypvOUZs6pIzQL4dJoiOURLb_O_pa0ucgLA3Eo7wZhRiWFIibxFiuzTaEOHKD-fiIWJvLRejRPhZ9ntZTMNtptLQ","p":"wKBy_OztSISyBSByV6jxTTWU4VIcXHeTAFhgfkv010FKuDceOcmMh6K9Q2nUUbhk0scGSrkCdDuvrQGOiYpgh_Ea_wkQHTb_WVVbzIYOZ4L495A5iuDTVGe3EunZYid7zOCH6FtWJZrZrYrL18Gv6r2hm5IuI-Mph7hlUMyuwo8","q":"4c-c-7WoWDTmO-Fbwq2fie4IpVmjRthoM7c1UZWmGl8fQQgly05pL13lg3RxPfT1xvN4xdFLICMj4cDZljr3CHFNWGUGX7R4GuswFomyIzsVwwm3Hpd3YFvsWLbQ0CvM2ZkQPw-qzbxqWkuA1cCtbGSJq41Vvz34WKE5k_J8QoM","qi":"BQcr66fwGnZPtLy4hr1gSkFBR_C7pTuxu3QmWTi_3yK2uLRZFCJiyWSuXvfWbT7Ns7jMugHWcN6eW8bBQ-DHEPHv8wbsWIigLTqBkZsE9aJDkhvgfgneWJpmC91wshkPcrI3TfoyCed6GyBgcb7_P7Ssi_rM25VavDQr5HrkYY0"}
```

Comment:
What is this file for?

Reply:
I don't know rly XD
It was auto generated

Reply:
It is developer signing key
![image](https://user-images.githubusercontent.com/77879337/142868070-be3bd8ab-2b7b-4e0c-b106-59f93ee052ef.png)

Reply:
@adrnchk 

As you can see, message said that you don't have to check this file into source code.

Reply:
ok, I will add it to gitignore